packages/context/README.md

```diff
@@ -0,0 +1,80 @@
+# @solid-primitives/context
+
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg?style=for-the-badge)](https://lerna.js.org/)
+[![size](https://img.shields.io/bundlephobia/minzip/@solid-primitives/context?style=for-the-badge&label=size)](https://bundlephobia.com/package/@solid-primitives/context)
+[![version](https://img.shields.io/npm/v/@solid-primitives/context?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/context)
+[![stage](https://img.shields.io/endpoint?style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fdavedbase%2Fsolid-primitives%2Fmain%2Fassets%2Fbadges%2Fstage-0.json)](https://github.com/davedbase/solid-primitives#contribution-process)
+
+Primitives simplifying or extending the SolidJS Context API.
+
+- **[`createContextProvider`](#createContextProvider)** - Create the context provider component & useContext function with types inferred from the factory function.
```

Comment:
```suggestion
- **[`createContextProvider`](#createContextProvider)** - Create a Context Provider component and useContext function, with types inferred from the factory function.
```

Reply:
Also, this is repeated verbatim below. Perhaps remove one of the duplicates?

Reply:
It's repeated on purpose, although it can look silly right now as there aren't any other primitives in this package. For example this is how it looks when there are more primitives: [event-listener#readme](https://github.com/davedbase/solid-primitives/tree/main/packages/event-listener#readme)
Maybe this outline or a primitive is not needed in listing...

Reply:
Maybe omit the outline until there are multiple primitives?